In attempt to find the cause of some of the hibernate issues we are seeing,
turn off threading for compression.

Signed-off-by: Justin M. Forbes <jforbes@redhat.com>

diff -ruNp a/kernel/power/swap.c b/kernel/power/swap.c
--- a/kernel/power/swap.c	2012-03-02 13:52:27.777485028 -0600
+++ b/kernel/power/swap.c	2012-03-02 14:05:20.168115129 -0600
@@ -401,7 +401,7 @@ static int swap_writer_finish(struct swa
 #define LZO_CMP_SIZE	(LZO_CMP_PAGES * PAGE_SIZE)
 
 /* Maximum number of threads for compression/decompression. */
-#define LZO_THREADS	3
+#define LZO_THREADS	1
 
 /* Maximum number of pages for read buffering. */
 #define LZO_READ_PAGES	(MAP_PAGE_ENTRIES * 8)
